src/core/actions/index.ts

```diff
@@ -206,6 +207,20 @@ namespace Actions {
   // added automatically by middleware to interact with redux-undo
   export const SAVE_STATE = 'SAVE_STATE';
   // tslint:enable max-line-length
+
+  // cart
+  export const GET_TRACKER_INFO = 'GET_TRACKER_INFO';
+  export type GetTrackerInfo = Action<typeof GET_TRACKER_INFO, Payload.Cart.TrackerInfo>;
+  export const CART_CREATED = 'CART_CREATED';
+  export type CartCreated = Action<typeof CART_CREATED, string>;
+  export const ADD_TO_CART = 'ADD_TO_CART';
+  export type AddToCart = Action<typeof ADD_TO_CART, Store.CartProduct>;
+  export const CART_SERVER_UPDATED = 'CART_SERVER_UPDATED';
+  export type CartServerUpdated = Action<typeof CART_SERVER_UPDATED, Partial<Store.Cart>>;
+  export const ITEM_QUANTITY_CHANGED = 'ITEM_QUANTITY_CHANGED';
+  export type ItemQuantityChanged = Action<typeof ITEM_QUANTITY_CHANGED, any>;
+  export const REMOVE_ITEM = 'REMOVE_ITEM';
+  export type RemoveItem = Action<typeof REMOVE_ITEM, Store.CartProduct>;
```

Comment:
Should maybe include the word "cart" with all the actions, like we do with other action types. The reason for this, is what if in the future there's a `Wishlist` api, which would be very similar to cart (needing the same actions, etc.) so we need a way of differentiating that this is the CartRemoveItem action, etc.